### ISSUE TITLE
ACME2 compatibility

### ISSUE THREAD

**Issue by Lekterx** [OptimalBits/redbird#256]
Hi there,

I've been using Redbird for a few month and I like it a lot, thank you for this !

My question comes a bit late, but it seems that Let's Encrypt certificates renewal will stop working starting on... Monday (06/01), due to the fact that my Redbird agent is not compatible with ACME2.

Since I may probably not be the only one receiving such warnings from Let's Encrypt, how did you all manage this situation ? Or am I missing something ?

Thx

**Comment by lezsakdomi** [comment]
I also just ran into this.

https://github.com/OptimalBits/redbird/issues/137 seems relevant.

**Comment by Lekterx** [comment]
Thanks a lot for your answer ! 

I actually saw #137 too, but sadly I don't really get how it helps...

**Comment by lezsakdomi** [comment]
I actually ended up installing certbot (https://certbot.eff.org/lets-encrypt/debianjessie-other), below is my proxy "configuration":

```javascript
#!/usr/bin/env node
const docker = require('redbird').docker;
const proxy = require('redbird')({
        port: 80,
        //letsencrypt: {
        //      path: __dirname + '/certs',
        //      port: 9999,
        //},
        ssl: {
                port: 443,
                //cert: '/etc/letsencrypt/'
        },
});

//const opts = {
//      ssl: {
//              //letsencrypt: {
//              //      email: 'domonkos.lezsak@weborigo.eu',
//              //      //production: false,
//              //      production: true,
//              //},
//      },
//};

proxy.register("domain1", "http://localhost:8080", {
        ssl: {
                cert: '/etc/letsencrypt/live/domain1/cert.pem',
                key:  '/etc/letsencrypt/live/domain1/privkey.pem',
        },
});
proxy.register("dev.domain1", "http://localhost:3000", {
        ssl: {
                cert: '/etc/letsencrypt/live/domain1/cert.pem',
                key:  '/etc/letsencrypt/live/domain1/privkey.pem',
        },
});
docker(proxy).register('domain2', 'registry.gitlab.com/weborigo/_/service2-front-end', {
        ssl: {
                cert: '/etc/letsencrypt/live/domain2/cert.pem',
                key:  '/etc/letsencrypt/live/domain2/privkey.pem',
        },
});
```

@Lekterx if you are lost, ping me

I think this issue can be closed in favor of #137

**Comment by Lekterx** [comment]
Thanks a lot for your answer !
